spec/features/deleting_a_user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.feature 'Finding a user' do
+  scenario 'successfully' do
+    sign_in_as_admin
+    @user = FactoryBot.create(:user)
+    stub_request(:post, 'https://testdomain/oauth/token')
+      .to_return(status: 200, body: '{"access_token":"TOKEN"}')
+    stub_request(:delete, "https://testdomain/api/v2/users/#{@user.auth_id}")
+      .to_return(status: 200, body: '')
+    visit admin_user_path(@user)
+    click_on 'Delete user'
+    click_on 'Delete user'
+    expect(page).to_not have_content(@user.name)
```

Comment:
I think this test would be less likely to raise false positives if it was testing something more specific, e.g. the presence of something on the page as there will be lots of pages that don't contain the user's name. given we're setting the flash alert, perhaps `expect(page).to have_content('User has been deleted')` instead.